app/src/components/integrations/elements/integrationSettings/connectionSection/connectionSection.jsx

```diff
@@ -200,45 +220,38 @@ export class ConnectionSection extends Component {
                   'connection-block-failed': !connected,
                 })}
               >
-                {!connected ? (
-                  <>
-                    {Parser(ErrorIcon)} <p>{formatMessage(messages.connectionFailedMessage)}</p>
-                  </>
-                ) : (
-                  <>
-                    {Parser(Tick)} <p>{formatMessage(messages.connectedMessage)}</p>
-                  </>
-                )}
+                {Parser(connected ? Tick : ErrorIcon)}{' '}
+                <p>
+                  {formatMessage(
+                    connected ? messages.connectedMessage : messages.connectionFailedMessage,
+                  )}
+                </p>
               </div>
             </div>
             {integrationParameters.url && (
               <div className={cx('sub-block')}>
-                <span>Link to BTS</span>
+                <span>{formatMessage(messages.linkTo)} BTS</span>
```

Comment:
Why you extract `BTS`? We have translation for it too f.e. `BtsCommonMessages.btsUrlHint` key in localizations